### PR TITLE
Protocol layer clarification

### PR DESCRIPTION
**Pull request by NateWilliams2**
This PR separates the census mechanism and client from the protocol layer

### CODE REVIEW

File: src/integration/census/general.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Census Integration
+
+The Voting Protocol requires a [Census Merkle Tree](../../architecture/census-overview.md) to be created and published before an organization can host a voting process. There are two main types of census, as well as two sub-types: 
++ [on-chain](on-chain.md) 
++ [off-chain](off-chain.md)
+  + [CSV](csv.md)
+  + [registry database](registry.md)
++ [off-chain registry database](registry.md).
```
Comment (ed255) [inline src/integration/census/general.md:8]: The previous line already refers to `regystry.md`.

**Review by ed255 — CHANGES_REQUESTED** [review]

**Review by p4u — COMMENTED** [review]
Regarding the census, I think we should split it on two main sections:

- Off chain based census (merkle tree)
- On chain based census (ethereum erc20)

Here some details that might be reused: https://www.notion.so/aragonorg/Introducing-Vocdoni-Bridge-cf7e73d38c4a45788358e9a1497cdf19

**Comment by ed255** [comment]
I see that these three files are being deleted:
- `src/manager/manager-api.md`
- `src/manager/overview.md`
- `src/manager/push-notifications-api.md`
- `src/manager/registry-api.md`
These files contain explanations about the funcionality of the manager and the details of its API.  Could you briefly explain why are these files being deleted, so that I can understand this change in context?  Thanks!

In `src/architecture/services/census-service.md` now there's a warning saying that the section is being deprecated.  But I understand we still use a census service in our product.  Why is this section being deprecated?  Will it be removed completely?  Or moved to some place else?  Thanks!

**Comment by p4u** [comment]
The Manager does not exist anymore (its deprecated). We want to rescue it it some moment but we prefer to have in the current documentation only those things which are currently working.

**Comment by NateWilliams2** [comment]
@ed255 I was under the impression that the census service was no longer used, and that the current frontend does all the census operations itself. Is this wrong? cc @p4u

**Comment by p4u** [comment]
Ah wow, I did not saw that. Yes the census service is being used, it's part of the Gateway. I was talking about the registry API (which is actually deprecated for now).

**Comment by NateWilliams2** [comment]
@p4u I see. So, after generating the Merkle tree from the CSV, the census service is used to publish the tree and generate the proofs themselves?

**Comment by p4u** [comment]
The CSV Merkle Tree is generated using the census-service of the gateways. The CSV only changes the origin of the information, it can be either a private database (as it was before with the Manager) or a CSV file provided directly by the user.

**Review by ed255 — APPROVED** [review]

**Comment by p4u** [comment]
@brickpop was also working on the kind of census descriptions, in separated document. Maybe he could share the link and also review the census section.

**Comment by NateWilliams2** [comment]
plz review again if possible @p4u  :)

File: src/integration/census/csv.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+The CSV census enables organizations to host voting processes without requiring users to store their own private keys, best used for a light-weight frontend with minimal user interaction. The entity generates a CSV file including user information. For example:
+
+| name     | age | country     | favorite color |
+| -------- | --- | ----------- | ----------     |
+| John     | 27  | Winterfell  | red            |
+| Tyrion   | 36  | Lannisport  | blue           |
+| Daenerys | 22  | Dragonstone | fuchsia        |
+| Jorah    | 65  | Bear Island | brown          |
+
+This information is normalized, concatenated, and then hashed to create a public key for each user. The public keys are added to the census Merkle Tree and then discarded, and only the Merkle Tree is stored. Users can then generate their ephemeral private key in the future by entering their correct information in a web client.
```
Comment (p4u) [inline src/integration/census/csv.md:16]: The information is also salted with the processId

**Review by p4u — APPROVED** [review]
LGTM, great work!

**Pull request merged**